pkg/cmd/pipelinerun/logs_test.go

```diff
@@ -2164,7 +2455,26 @@ func TestLog_pipelinerun_only_one_v1beta1(t *testing.T) {
 	test.AssertOutput(t, prName, lopt.PipelineRunName)
 }
 
-func logOpts(name string, ns string, cs pipelinetest.Clients, dc dynamic.Interface, streamer stream.NewStreamerFunc, allSteps bool, follow bool, tasks ...string) *options.LogOptions {
+func logOptsv1aplha1(name string, ns string, cs pipelinetest.Clients, dc dynamic.Interface, streamer stream.NewStreamerFunc, allSteps bool, follow bool, tasks ...string) *options.LogOptions {
+	p := test.Params{
+		Kube:    cs.Kube,
+		Tekton:  cs.Pipeline,
+		Dynamic: dc,
+	}
+	p.SetNamespace(ns)
+
+	logOptions := options.LogOptions{
+		PipelineRunName: name,
+		Tasks:           tasks,
+		AllSteps:        allSteps,
+		Follow:          follow,
+		Params:          &p,
+		Streamer:        streamer,
+	}
+
+	return &logOptions
+}
+func logOptsv1beta1(name string, ns string, cs pipelinev1beta1test.Clients, dc dynamic.Interface, streamer stream.NewStreamerFunc, allSteps bool, follow bool, tasks ...string) *options.LogOptions {
```

Comment:
Any reason there needs to be two separate funcs here instead of one that takes a version param?